packages/ddd/src/generators/init/schema.json

```diff
@@ -0,0 +1,8 @@
+{
+  "$schema": "https://json-schema.org/schema",
+  "cli": "nx",
+  "$id": "init",
+  "title": "Add ESLint rules for enforce-module-boundaries",
+  "type": "object",
+  "properties": {}
```

Comment:
Currently, no options are needed here.

Reply:
why? please be more descriptive, just to remind you of the way of setting global options for schematics, like for angular component to generate only onPush components:
in angular/workspace.json you have schematics part e.g
```JSON
{
  ...,
  "schematics": {
    "@nrwl/angular:component": {
      "style": "scss",
      "changeDetection": "OnPush"
    }
  }
}
```

so that's something that can be done in you init, to set the name of the shared domain, to set defaults or whatever